File: ConsorcioGestBack/BusinessService/Models/UserModel.cs
using BusinessService.Models.AuxModel;
using DataAccess.Data.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BusinessService.Models
{
    public class UserModel
    {
        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public string LastName { get; set; } = null!;
        public string? Phone { get; set; }
        public string Email { get; set; } = null!;
        public bool IsOwner { get; set; }
        public bool IsOcupant { get; set; }
        public int? IdCondominium { get; set; }
        public string Condominio { get; set; }
        public ProfileModel Profile { get; set; } = null;
        public StateModel UserState { get; set; } = null;
        public int? IdDocumentType { get; set; }
        public int Document { get; set; }
        public string Token { get; set; }
    }
}

File: ConsorcioGestFront/ConsorcioGestWeb/src/app/Models/Models/UserModel.ts
export class UserModel {
    id: number = 0;
    name: string = '';
    lastName: string = '';
    phone?: string;
    email: string = '';
    isOwner: boolean = false;
    isOcupant: boolean = false;
    idCondominium?: number;
    condominio:string = '';
    profile: ProfileModel = new ProfileModel();
    userState: StateModel = new StateModel();
    idDocumentType?: number;
    document: number = 0;
    token: string = '';
}

export class ProfileModel {
    id: number = 0;
    name: string = '';
}

export class StateModel {
    id: number = 0;
    name: string = '';
}
